src/find-specs.js

```diff
@@ -43,6 +45,8 @@ function canonicalizeTRUrl(url) {
 
 const toGhUrl = repo => `https://${repo.owner.login.toLowerCase()}.github.io/${repo.name}/`
 const matchRepoName = fullName => r => fullName === r.owner.login + '/' + r.name;
+const isNotIgnorableRepo = fullName => !Object.keys(ignored.repos).includes(fullName) && !Object.keys(monitoredRepos).includes(fullName);
+const isNotIgnorableSpec = url => !Object.keys(ignored.specs).includes(url);
```

Comment:
Editorial: That looks like a double negative. I would rename to something like `isRelevantRepo` and `isRelevantSpec`. Always stay positive in life as they say!